docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/controller-was-group-proc.adoc

```diff
@@ -0,0 +1,22 @@
+[id='controller-was-goup-proc_{context}']
+= Creating and mapping the `controller` role
+
+After {KIE_SERVER} is installed, you must create the `kie-server` role and `kie-server` group, then map the `kie-server` role to the `kie-server` group before you run {KIE_SERVER}.
+
+.Prerequisite
+* {KIE_SERVER} is installed on the {WEBSPHERE} instance.
+* {KIE_SERVER} nodes contain a user with `kie-server` role.
+* Headless {PRODUCT_SHORT} controller nodes contain a user with `kie-server` role.
+
+
+.Procedure
+. In the WebSphere Integrated Solutions Console, to create the `kie-server` group, click *Users and Groups* -> *Manage Groups*.
```

Comment:
Maybe lose the "to create the `kie-server` group" part in the first step, since you don't actually create the kie-server group until step 3? Alternatively, have it at the beginning of the step? 

Up to you, this step just feels a little busy to me.